contracts/lido_terra_hub/src/testing/mock_querier.rs

```diff
@@ -127,6 +133,36 @@ impl WasmMockQuerier {
                     validators.sort_by(|v1, v2| v1.total_delegated.cmp(&v2.total_delegated));
                     return SystemResult::Ok(ContractResult::from(to_binary(&validators)));
                 }
+                if contract_addr == AIRDROP_REGISTRY {
+                    if let QueryAirdropRegistry::AirdropInfo {
+                        airdrop_token,
+                        start_after: _,
+                        limit: _,
+                    } = from_binary(msg).unwrap()
+                    {
+                        if airdrop_token.clone().unwrap() == AIRTOKEN {
+                            return SystemResult::Ok(ContractResult::from(to_binary(
+                                &AirdropInfoResponse {
+                                    airdrop_info: vec![AirdropInfoElem {
+                                        airdrop_token: AIRTOKEN.to_string(),
+                                        info: basset::airdrop::AirdropInfo {
+                                            airdrop_token_contract: AIRDROP_TOKEN_CONTRACT
+                                                .to_string(),
+                                            airdrop_contract: AIRDROP_CONTRACT.to_string(),
+                                        },
+                                    }],
+                                },
+                            )));
+                        } else {
+                            return SystemResult::Err(SystemError::InvalidRequest {
+                                error: format!("token not found: {}", airdrop_token.unwrap()),
+                                request: msg.as_slice().into(),
+                            });
+                        }
+                    } else {
+                        todo!()
```

Comment:
I am not sure if we are doing an intermediate or final review. If we are at final review, this todo doesn't look good.

Reply:
just a IDE code generation, removed the branch